src/components/HomepageFeatures/index.js

```diff
@@ -1,64 +1,142 @@
+// REFER TO THIS SECTION TO CONTRIBUTE TO THIS FILE
+
+// Here 
+
+// 1. hp refers to Homepage
+// 2. cont refers to Container
+// 3. p refers to Paragraph
+
+// For Example :-
+
+// (hp_cont_3_btn2_content) refers to "Content present in Button number 2  of Container number 3 on Homepage"
+
+// File for homepage
+
 import React from 'react';
-import clsx from 'clsx';
+
+// Imported CSS file from "./styles.module.css"
+
 import styles from './styles.module.css';
 
-const FeatureList = [
-  {
-    title: 'Easy to Use',
-    Svg: require('@site/static/img/undraw_docusaurus_mountain.svg').default,
-    description: (
-      <>
-        Docusaurus was designed from the ground up to be easily installed and
-        used to get your website up and running quickly.
-      </>
-    ),
-  },
-  {
-    title: 'Focus on What Matters',
-    Svg: require('@site/static/img/undraw_docusaurus_tree.svg').default,
-    description: (
-      <>
-        Docusaurus lets you focus on your docs, and we&apos;ll do the chores. Go
-        ahead and move your docs into the <code>docs</code> directory.
-      </>
-    ),
-  },
-  {
-    title: 'Powered by React',
-    Svg: require('@site/static/img/undraw_docusaurus_react.svg').default,
-    description: (
-      <>
-        Extend or customize your website layout by reusing React. Docusaurus can
-        be extended while reusing the same header and footer.
-      </>
-    ),
-  },
-];
-
-function Feature({Svg, title, description}) {
-  return (
-    <div className={clsx('col col--4')}>
-      <div className="text--center">
-        <Svg className={styles.featureSvg} role="img" />
-      </div>
-      <div className="text--center padding-horiz--md">
-        <h3>{title}</h3>
-        <p>{description}</p>
-      </div>
-    </div>
-  );
-}
+// Imported all the images below
+
+import comments from './assests/Comments.png'
+import reactions from './assests/Reactions.png'
+import computer from './assests/Computer.png'
+import githubActions from './assests/github-actions.png'
+import homepageVid from './assests/Videos.png'
+import homepageBlog from './assests/Blogs.png'
```

Comment:
We have imported this but not used in the file do we need this

Reply:
Really sorry forgot to remove those 2 imports after using them in CSS.